src/debug.cpp

```diff
@@ -524,6 +524,16 @@ void realDebugmsg( const char *filename, const char *line, const char *funcname,
         return;
     }
 
+    if( isDebuggerActive() ) {
+#if defined(_WIN32)
+        DebugBreak();
+        return;
+#elif defined( __linux__ )
+        raise( SIGTRAP );
+        return;
+#endif
```

Comment:
This can trigger needlessly when there are debug messages that the programmer do not care about. Debug messages do not necessarily crash the game so I think the programmer can add the breaking point themselves if they want to. Only crash need to be handled differently.